### PR TITLE
db traduzido

### PR DESCRIPTION
**Pull request by Pyri47**
traduzi para inglês.

### CODE REVIEW

File: BaseDados/Código/Readme [hunk 1]
```diff
@@ -1,42 +1,42 @@
...
-descricao varchar(50) );
+description varchar(50) );
 
-INSERT INTO tipo_imovel (descricao) VALUES
-	('Garagem'),
-	('Quarto'),
-	('Moradia'),
-	('Apartamento'),
-	('Estudio');
+INSERT INTO realstate_type (description ) VALUES
```
Comment (jonathan-osorio) [inline BaseDados/Código/Readme:17]: Change realstate for realestate (in the other tables as well)

**Review by jonathan-osorio — COMMENTED** [review]
Just a quick spelling fix, translating the commit and PR name and creating a new README for the translated version.

**Comment by pmzpro** [comment]
> traduzi para inglês.

> *translated to english.

File: BaseDados/Código/DB_English.txt [hunk 2]
```diff
@@ -96,18 +96,18 @@ CREATE TABLE realstate (
...
 	fk_typology_id INT
 );
 
-ALTER TABLE realstate_has_amenities 
-	ADD CONSTRAINT fk_realstate_id FOREIGN KEY (fk_realstate_id) REFERENCES realstate(realstate_id),
+ALTER TABLE realestate_has_amenities 
+	ADD CONSTRAINT fk_realestate_id FOREIGN KEY (fk_realestate_id) REFERENCES realestate(realestate_id),
 	ADD CONSTRAINT fk_amenities_id FOREIGN KEY (fk_amenities_id) REFERENCES amenities(id);
 
 ALTER TABLE imovel
```
Comment (pmzpro) [inline BaseDados/Código/DB_English.txt:108]: You forgot to change the name of the table on the ALTER. `imovel` fix to `ALTER TABLE realestate`.

**Review by pmzpro — COMMENTED** [review]
Line 108 should be fixed.

**Pull request closed without merging**